static/templates/no_js.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width,initial-scale=1" />
     <link type="root" href="{{root}}">
-    <title>Welcome to Kiwix Server</title>
+    <title>{{welcome-to-kiwix-server}}</title>
     <link
       type="text/css"
       href="{{root}}/skin/index.css?KIWIXCACHEID"
```

Comment:
Passing translation strings as top-level mustache variables feels somewhat fragile to me. Can't all translations required for this template be passed as one `translations` object (so that they are accessed as `{{translations.welcome-to-kiwix-server}}`? I am not sure that it will work but I believe that it should.

Reply:
Done. Thanks